user-guide/Cloud_Platform/CloudAdminApp/Granting_admin_consent.md

```diff
@@ -4,10 +4,13 @@ uid: Granting_admin_consent
 
 # Granting admin consent for Teams actions
 
-When the Microsoft Teams Chat Integration functionality is used, admin consent is required for certain actions, e.g. to create teams or channels, or to add members to a team.
+To use the [Microsoft Teams Chat Integration](xref:Microsoft_Teams_Chat_Integration) functionality, which involves DataMiner Automation interacting with Teams without user interaction, admin consent is required. This is specifically needed for actions such as creating teams or channels, or adding members to a team.
```

Comment:
Maybe also add other features (sending channel/chat notifications etc) or link to the ChatIntegration page where a list of these: https://docs.dataminer.services/user-guide/Cloud_Platform/TeamsBot/Microsoft_Teams_Chat_Integration.html#using-chat-integration